packages/@react-spectrum/form/stories/Form.stories.tsx

```diff
@@ -689,6 +690,12 @@ function FormWithSubmit() {
 
   return (
     <Form onSubmit={handleSubmit} isReadOnly={formStatus === 'valid'}>
+      {(formStatus === 'invalid' || formStatus === 'valid') &&
+        <InlineAlert variant={formStatus === 'invalid' ? 'negative' : 'positive'}>
+          <Header>{formStatus === 'invalid' ? 'Error' : 'Success'}</Header>
```

Comment:
Only "concern" is that when the Header text is the same as the variant string it says the same word twice. That is an issue for the developing team... wondering if we'll get complaints.

Reply:
Yeah, I think apps should hopefully use more specific titles.